cylc/flow/simulation.py

```diff
@@ -91,6 +139,11 @@ def get_simulated_run_len(rtc: Dict[str, Any]) -> int:
             str(rtc['simulation']['default run length'])
```

Comment:
s/get_simulated_run_len/set_simulated_run_len

Reply:
> This now sets the sim time

Where?

Reply:
Where does it set the sim_time?

Reply:
It sets the execution time limit a couple of lines down.

On the whole we would expect a `get_` method to return a value, but not to make changes to things.

Reply:
The `execution time limit` is not the simulation run time though. It's just a side effect.